duckduckgo/macos-browser
# Update dashboard

Task/Issue URL: https://app.asana.com/0/0/1202675340331509/f
Tech Design URL:
CC:

**Description**:
This PR updates the privacy dashboard to say **Cookies Managed** instead of **Cookies Minimized**.



**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/features/autoconsent/
1. Open the dashboard. At the bottom it should say "Cookies Managed" **not** "Cookies Minimized"

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
